core/ErrorFlusherStdout.cc

```diff
@@ -73,4 +66,12 @@ void ErrorFlusherStdout::flushAutocorrects(const GlobalState &gs, FileSystem &fs
     autocorrects.clear();
 }
 
+void ErrorFlusherStdout::flushErrorCount(spdlog::logger &logger, int count) {
```

Comment:
(suuuuuuuuuper minor nit) To avoid dirtying the diff you can move this back to its old position ^^